smoke/LeafDevice/details/Details.cs

```diff
@@ -64,41 +131,63 @@ public class Details
             }
         }
 
-        protected Task InitializeServerCerts()
```

Comment:
Nit: might be cleaner to have 2 separate methods - one for server certs and one for client certs.

Reply:
At the end, they are both simply trusted certificates that need to get installed and both can have different roots. I felt generalizing this simplified things and also in line with our existing terminology.